docs/sources/datasources/jaeger.md

```diff
@@ -39,6 +39,12 @@ This is a configuration for the [trace to logs feature]({{< relref "../explore/t
 
 ![Trace to logs settings](/static/img/docs/explore/trace-to-logs-settings-8-2.png 'Screenshot of the trace to logs settings')
 
+### Node Graph
+
+This is a configuration for the Node Graph visualization.
+
+-- **Enable Node Graph -** Enables the Node Graph visualization.
```

Comment:
Could we add a bit more info here and include that:
- the graph is shown after the trace view is loaded
- it's disabled by default
- it's a beta functionality